tests/test_xkcdpass.py

```diff
@@ -51,8 +51,9 @@ def test_delim(self):
     def test_random_delimiter(self):
         wordlist = xkcd_password.generate_wordlist(WORDFILE, min_length=3, max_length=3)
         result = xkcd_password.generate_xkcdpassword(wordlist, numwords=3, random_delimiters=True)
+        answer = len(result)
 
-        self.assertEqual(11, len(result))
+        self.assertTrue(answer == 11 or answer == 10 or answer == 9)
```

Comment:
I'm not a python developer, but can't we do something like this:

```py
self.assertTrue(len(result) in [ 9, 10, 11 ])
```

Reply:
Thanks for the fix! I implemented @peterhoeg's suggested alternative syntax in the end, I haven't been able to trigger the random test failure. I'll leave the PR open for noww - once you're happy its working on Hydra let me know and I can close it off

Reply:
I recommend using a Set instead of a List.
```
self.assertTrue(len(result) in {9, 10, 11})
```
or
```
self.assertTrue(len(result) in set(range(9,12)))
```

Reply:
Good suggestion, I have updated the code to use your first option

Reply:
Personally I recommend to re-write the assert to embrace the desired consequences of the `random_delimiters` argument:

```python
assert any(d in result for d in DEFAULT_DERIMITERS)
```

Or, imagine any other assertion that prove that result contains from 1 to 3 delimiters from default set (which, I hope, immediately take you to conclusion that an empty string, you know, is not "delimiter" of any kind).

Reply:
I've updated the test (and the delimiter list - good catch) with this feedback: https://github.com/redacted/XKCD-password-generator/blob/90a717c8cb70394a40127366b3739edbaa82f1fa/tests/test_xkcdpass.py#L51. It now checks that both the final output length and number of delimiter characters are as expected. As the empty string is removed from the possibilities overall length should be deterministic